obstacle_avoidance_node:
  ros__parameters:
    avoidance_lieaner: 0.8
    avoidance_angular: 1.2
    wall_threshold: 0.35